x-pack/plugins/ingest_manager/public/applications/ingest_manager/hooks/use_core.ts

```diff
@@ -4,15 +4,13 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { useContext } from 'react';
-import { CoreStart } from 'src/core/public';
+import { CoreStart } from 'kibana/public';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
 
-export const CoreContext = React.createContext<CoreStart | null>(null);
-
-export function useCore() {
-  const core = useContext(CoreContext);
-  if (core === null) {
-    throw new Error('CoreContext not initialized');
+export function useCore(): CoreStart {
```

Comment:
updated useCore() to use the new useKibana hook

Reply:
This is pretty cool, I didn't know that `kibana_react` had these kind of components/hooks now